app/models/notifications/queries/RoeCode.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.notifications.queries
+
+import play.api.Logger
+import play.api.libs.json.{Format, JsResult, JsString, JsSuccess, JsValue}
+
+sealed trait RoeCode extends Ordered[RoeCode] {
+  val code: String
+  val priority: Int
+
+  override def compare(that: RoeCode): Int = this.priority - that.priority
+}
+
+object RoeCode {
+  val logger = Logger(this.getClass)
+
+  case object PhysicalExternalPartyControl extends RoeCode {
+    val code = "2"
+    val priority = 1
+  }
+
+  case object DocumentaryControl extends RoeCode {
+    val code = "1"
+    val priority = 2
+  }
+
+  case object NonBlockingDocumentaryControl extends RoeCode {
+    val code = "3"
+    val priority = 3
+  }
+
+  case object NoControlRequired extends RoeCode {
+    val code = "6"
+    val priority = 4
+  }
+
+  case object UnknownRoe extends RoeCode {
+    val code = ""
+    val priority = 5
```

Comment:
Here you can easily go with for e.g. 100. Won't need to rewrite it every time you add new code.